wpeverest/everest-forms
# Enhancement - Send file directly as an attachment.

### All Submissions:






### Changes proposed in this Pull Request:


Previously When we enable Disable storing entry information. The file upload field is stored in server and link is provided in email.This PR solves this issue.

### How to test the changes in this Pull Request:

1. Create a form with file upload.
2. Go to > Settings > General > Disable storing entry information.
3.  Submit the form and check the file uploaded will send as attachment.
4.  The uploaded file will no stored in server. 

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Send file directly as an attachment. .
